server/groups/class.lua

```diff
@@ -5,6 +5,9 @@
 ---@field principal string
 ---@field hasAccount boolean
 ---@field adminGrade number
+---@field type = string,
+---@field type_label = string,
```

Comment:
type_label either not useful (above) or should be typeLabel (camel casing).
unique_type should be uniqueType, isUnique, or such.